readthedocs.yml

```diff
@@ -10,7 +16,6 @@ sphinx:
   fail_on_warning: true
 
 python:
-   version: 3.8
```

Comment:
Python version is governed by conda.

Reply:
Doesn't this make RTD setup a conda env with this version? If not should we put the python version in the rtd-env file so we are explicit about it?

Reply:
I can add it.